kafka-streams-partitioners/weighted-group-partitioner/src/main/java/org/hypertrace/core/kafkastreams/framework/partitioner/PartitionerConfigServiceClientConfig.java

```diff
@@ -0,0 +1,35 @@
+package org.hypertrace.core.kafkastreams.framework.partitioner;
+
+import com.typesafe.config.Config;
+import java.time.Duration;
+import lombok.Builder;
+import lombok.Value;
+
+@Builder
+@Value
```

Comment:
nit: adding a `@Builder` gives a bit more flexibility to consumers on how they want to use this - whether they hardcode it, manage the config or use the predefined format.